test/support/transition_landing_page_steps.rb

```diff
@@ -6,38 +6,15 @@ module TransitionLandingPageSteps
   include GdsApi::TestHelpers::ContentItemHelpers
   include RummagerHelpers
 
-  def given_there_is_a_brexit_taxon
-    stub_content_store_has_item(brexit_taxon_path, content_item)
-  end
-
-  def brexit_taxon_path
-    "/brexit"
-  end
+  CONTENT_ID = "d6c2de5d-ef90-45d1-82d4-5f2438369eea".freeze
+  TRANSITION_TAXON_PATH = "/transition".freeze
 
-  def content_id
```

Comment:
Or perhaps we resurrect a method like this called `transition_taxon_id` and just shove it in there?